homeassistant/components/light/lifx/__init__.py

```diff
@@ -434,9 +435,7 @@ def find_hsbk(self, **kwargs):
         if hsbk is not None:
             return [hsbk, True]
 
-        color_name = kwargs.pop(ATTR_COLOR_NAME, None)
-        if color_name is not None:
-            kwargs[ATTR_RGB_COLOR] = color_util.color_name_to_rgb(color_name)
+        preprocess_turn_on_alternatives(kwargs)
```

Comment:
This is because LIFX effects (and also `lifx_set_state` now) accept the same parameter names as `light.turn_on`, without actually going through the `turn_on` handler.

I previously wrote it out for `color_name`, but now with four alternative names I thought it to be too much duplication.